tests/parsers/gdrive_synclog.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+"""Tests for the GoogleDriveSyncLog log parser."""
+
+from __future__ import unicode_literals
+
+import unittest
+
+from plaso.formatters import gdrive_synclog as _  # pylint: disable=unused-import
+from plaso.lib import timelib
+from plaso.parsers import gdrive_synclog
+
+from tests import test_lib as shared_test_lib
+from tests.parsers import test_lib
+
+# TODO: more tests, esp. for confirming Unicode support, other OS discrepancies.
+
+class GoogleDriveSyncLogUnitTest(test_lib.ParserTestCase):
+  """Tests for the Google Drive Sync client log parser."""
+
+  # TODO: it seems like this takes a different call path entirely when parsing
```

Comment:
Not sure what you mean with this comment, please clarify

Reply:
Removed; this may have been an oversight/misunderstanding on my part.

It looked like _ParseFile() was taking a different execution path than the non-test code was because some test cases passed when they failed if I ran log2timeline locally, but I think it was related to the BUFFER_SIZE change we had to make. Either the fake doesn't trigger it, or the test cases I manually added to get passing were too short to trigger it and I confused myself with the u''-string red herring.